templates/default/authinfo.ses.erb

```diff
@@ -1,2 +1,2 @@
-AuthInfo:email-smtp.<%= @aws_region %>.amazonaws.com "U:root" "I:<%= @username %>" "P:<%= @password %>" "M:LOGIN"
-AuthInfo:ses-smtp-prod-335357831.<%= @aws_region %>.elb.amazonaws.com "U:root" "I:<%= @username %>" "P:<%= @password %>" "M:LOGIN"
+AuthInfo:<%= @host %> "U:root" "I:<%= @username %>" "P:<%= @password %>" "M:LOGIN"
```

Comment:
Should be. The only catch is you encounter the bug that requires the ses-smtp-prod-335357831. But that section doesn't work right anyways right now, because ses-smtp-prod-335357831 is for all regions.